res/database/Default/database_ads.xml

```diff
@@ -422,32 +423,32 @@ But only with Relocation Rudolph!</en>
 				<en>Jesse water and heat services</en>
 			</title>
 			<description>
-				<de>Wir sorgen für ihre Sanitäranlagen. Schon ab 10 € - ohne Scheiß!</de>
-				<en>We take care of your sanitary facilities. Beginning from 10 € - no shit!</en>
+				<de>Wir sorgen für Ihre Sanitäranlagen. Schon ab 10 € - ohne Scheiß!</de>
+				<en>We take care of your sanitary facilities. Starting at 10 € - no shits!</en>
 			</description>
 			<conditions min_audience="0.4679" min_image="7" target_group="8" pro_pressure_groups="0" contra_pressure_groups="0"/>
 			<data quality="9" repetitions="3" duration="3" profit="2111" penalty="3280" infomercial_profit="119" fix_infomercial_profit="1"/>
 		</ad>
 		<ad id="7a86d4cd-7266-4247-ac6a-b63a62d12c93">
 			<title>
 				<de>Hobelmarkt</de>
-				<en>Plane-Market</en>
+				<en>Planer Market</en>
 			</title>
 			<description>
 				<de>Hier fallen nicht nur Späne, nein auch die Preise. Ein Hobelmarkt ist sicher auch in Ihrer Nähe.</de>
-				<en>It's not only chips dropping down but also the prices. A Plane-Market is certainly also near you.</en>
+				<en>Here not only the woodchips are dropping but also the prices. A Planer Market is certainly also in your area.</en>
 			</description>
 			<conditions min_audience="2.6" min_image="7" target_group="8" pro_pressure_groups="0" contra_pressure_groups="0"/>
 			<data quality="12" repetitions="4" duration="3" profit="750" penalty="1200" infomercial_profit="72" fix_infomercial_profit="1"/>
 		</ad>
 		<ad id="7bf56c92-52f5-44aa-8cef-2788ece421ac">
 			<title>
 				<de>Weißer Friese</de>
-				<en>White Friesian</en>
+				<en>White Frieze</en>
```

Comment:
Frieze = horse
Frisian = a person from Frisia

Reply:
It's also a sort of cloth, hence it was a somewhat good fit in this context.

https://en.wikipedia.org/wiki/Frieze_(textile)

The thing is that the original isn't applicable at all anyway, the brand doesn't exist internationally (Weißer Riese), nor can the rhyme be used (Weißer Friese). The translation surely is just unfunny, but it's not always possible to come up with something original. Maybe someone has a better idea.

Reply:
Not a biggy, keep it "Frieze" then